.circleci/config.yml

```diff
@@ -2,10 +2,10 @@
 version: 2
 
 x-config:
-  - &name "frogpond/ccc-server"
+  - &name 'frogpond/ccc-server'
   - &defaults
     docker:
-      - image: &default-image docker.io/circleci/node:10
+      - image: &default-image docker.io/circleci/node:12
```

Comment:
Should this be 18, too?

Reply:
No such image exists — I think this is a debian version or something?